app/Http/Controllers/V2/Terrafund/TerrafundEditGeometryController.php

```diff
@@ -188,16 +188,35 @@ public function updateGeometry(string $uuid, Request $request)
             }
             $geometry = json_decode($request->input('geometry'));
             $geom = DB::raw("ST_GeomFromGeoJSON('" . json_encode($geometry) . "')");
-            $polygonGeometry->geom = $geom;
-            $polygonGeometry->save();
+
             $sitePolygon = SitePolygon::where('poly_id', $polygonGeometry->uuid)->first();
-            if ($sitePolygon) {
-                $this->updateEstAreainSitePolygon($polygonGeometry, $geometry);
-                $this->updateProjectCentroidFromPolygon($polygonGeometry);
-                $sitePolygon->changeStatusOnEdit();
-            }
 
-            return response()->json(['message' => 'Geometry updated successfully.', 'geometry' => $geometry, 'uuid' => $uuid]);
+            $user = User::isUuid(Auth::user()->uuid)->first();
+            if (isset($request['adminUpdate']) && filter_var($request['adminUpdate'], FILTER_VALIDATE_BOOLEAN)) {
```

Comment:
Remove this parameter and verify if the User that is currently logged in is an admin.